src/patternfly/_chart-globals.scss

```diff
@@ -50,3 +51,189 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+// typography
+$pf-chart-global--FontSize--xs: 12; // Axis Labels
```

Comment:
Are we going to stick with unitless numbers or add units so it's clear what this is supposed to convey? The plan was that this stylesheet should also serve as the base styles for someone building a charting library that doesn't use victory.

Reply:
I would prefer that we stick with unitless numbers. We don't currently have a solution for react-tokens to prune units out

Reply:
Since it's all built out now, I talked to @dgutride and we think it's going to be difficult to position this as a generic stylesheet for other libraries to use, so seems fine to make it specific to victory.